c2corg_ui/static/partials/cards/outings.html

```diff
@@ -45,7 +45,7 @@
       </div>
     </div>
 
-    <div class="quality" ng-if="::doc['quality']" ng-class="::doc['quality']"
+    <div class="quality {{'icon'+ doc['quality']}}" ng-if="::doc['quality']" 
```

Comment:
I have tested in my instance. At least the generated HTML seems correct:
```
<div class="quality iconmedium" ng-if="::doc['quality']" uib-tooltip="Complétude : moyen" tooltip-append-to-body="true"> </div>
```

But the ``-`` is missing in the class name (should be ``icon-medium``). At the moment quality icons are not shown because of this typo.

Using ``ng-class`` is probably better though. But I can't spend time investigating now.

I think it is important that you carefully test your changes before updating the PR.